### ISSUE TITLE
WriteOption enum missing mongo-client 3.9.0

### ISSUE THREAD

**Issue by Dylancorbus** [reactiverse/es4x#335]
node_modules/@vertx/mongo-client/options.d.ts:68:35 - error TS2307: Cannot find module './enums'.

68 import { BulkOperationType } from './enums';
                                     ~~~~~~~~~

node_modules/@vertx/mongo-client/options.d.ts:129:29 - error TS2307: Cannot find module './enums'.

129 import { WriteOption } from './enums';
                                ~~~~~~~~~

**Comment by pmlopes** [comment]
Fixed by https://github.com/vert-x3/vertx-mongo-client/commit/e4b271f242326981541e1b4aa6502c6e9e04bf9f#diff-49bcf7b488f8b36e0c0b880421b0133b

**Comment by noelchavezsimbron** [comment]
@Dylancorbus  do yo solve this problem?   i'm had the same problem

**Comment by pmlopes** [comment]
@noelchavezsimbron you can already test the release candidates for 4.0.0. Also note that as vert.x is going to be release in the next weeks, this will be fixed once a official 4.0.0 is out.
